### PR TITLE
Make the hero units the same size as they are on Martsy

### PR DESCRIPTION
**Pull request by orta**
We have smaller hero units, I'm not sure if the spec changed since we first built out hero units, but admins expected to be able to add more information than we could comfortably show. This brings us back inline with martsy.

### CODE REVIEW

File: Artsy/View_Controllers/Core/ARHeroUnitViewController.m [hunk 1]
```diff
@@ -31,7 +31,7 @@ @implementation ARHeroUnitViewController
 
 + (CGFloat)heroUnitHeight
 {
-    return [UIDevice isPad] ? 380 : 232;
+    return [UIDevice isPad] ? 380 : 252;
```
Comment (alloy) [inline Artsy/View_Controllers/Core/ARHeroUnitViewController.m]: Is it working as expected on iPad?

**Comment by orta** [comment]
Before:
screen shot 2016-02-11 at 12 40 20

After:
screen shot 2016-02-11 at 12 47 55

File: Artsy/View_Controllers/Core/ARHeroUnitViewController.m (hunk 1 above)
Comment (orta) [inline Artsy/View_Controllers/Core/ARHeroUnitViewController.m]: Spacing wise? I think so, will give it a quick look

Comment (orta) [inline Artsy/View_Controllers/Core/ARHeroUnitViewController.m]: Yeah, they're gonna be fine.

**Comment by alloy** [comment]
Groovy

**Pull request merged**